src/main/java/org/broadinstitute/consent/http/service/ConsentService.java

```diff
@@ -339,12 +339,12 @@ public String getConsentDulUrl(String consentId) throws UnknownIdentifierExcepti
      **/
     private List<String> updateAssociations(String consentId, String associationType, List<String> ids) {
         Handle h = jdbi.open();
-        PreparedBatch insertBatch = h.prepareBatch("insert into consentassociations (consentId, associationType, dataSetId) values (?, ?, ?)");
+        PreparedBatch insertBatch = h.prepareBatch("insert into consent_associations (consent_id, association_type, data_set_id) values (?, ?, ?)");
```

Comment:
normalize SQL string? `INSERT INTO consent_associations (consent_id, association_type, data_set_id) VALUES (?, ?, ?)`

Reply:
good catch! I completely forgot to consider the SQL in this file as I was so focused on the actual DAO classes.